unstract/connectors/src/unstract/connectors/databases/bigquery/bigquery.py

```diff
@@ -18,7 +18,9 @@
 
 logger = logging.getLogger(__name__)
 
-BIG_QUERY_TABLE_SIZE = 3
+BIG_QUERY_TABLE_SIZE = (
+    3  # BigQuery table format 'database.schema.table' splits into an array of size 3
+)
```

Comment:
Why this in bracket? if only due to the long comment, move the comment just above the line.